### PR TITLE
Add Learn by Example site

### PR DESCRIPTION
**Pull request by JorgeCastilloPrz**
Just a WIP to create the menu and early list of stubbed sections. For accessing once deployed: http://127.0.0.1:4000/learn-by-example/

I copied the core module CSS and images to be able to render the site. However, the initial purpose is only the content (menu and pages). This would need to be iterated to have proper icon and colors, among other required things.

Screenshot 2020-03-31 at 12 46 42

Pending task: talk to design team about it.

cc: @rachelcarmena

### CODE REVIEW

File: docs/_sass/components/docs/_doc-content-learn-by-example.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// DOC CONTENT CORE
+// -----------------------------------------------
+// -----------------------------------------------
+
+.doc-content-learn-by-example {
```
Comment (raulraja) [inline docs/_sass/components/docs/_doc-content-learn-by-example.scss:5]: @israelperezglez should this CSS live here or should it be elsewhere?

Comment (rachelcarmena) [reply docs/_sass/components/docs/_doc-content-learn-by-example.scss:5]: We don't have the definitive style for this new section in the website. We just copied the styles from Arrow Core. It's just a way to preview this new section in the website. We'll change it as soon as we have the new styles :+1:

Comment (rachelcarmena) [reply docs/_sass/components/docs/_doc-content-learn-by-example.scss:5]: It happens the same with #32. We're focused on the content while styles are being built by design team.

Comment (JorgeCastilloPrz) [reply docs/_sass/components/docs/_doc-content-learn-by-example.scss:5]: Design is expected to work over this PR to polish / update design with a new logo and colors. They can decide where to put css at that point, but this is like any other arrow docs sites, their styles are already within this repository, which kind of acts like a coordination one.

Comment (israelperezglez) [reply docs/_sass/components/docs/_doc-content-learn-by-example.scss:5]: @raulraja Yeah, we could use that scss ;)

**Pull request closed without merging**